benchmarks/01_trivial/README.md

```diff
@@ -6,3 +6,4 @@
 - ``04_fastdiv`` - integer division by a constant is handled by an algorithm, which produces a code sequence without actual division operation. Current implementation relies on code generator to do the right thing on every specific platform. This benchmark tests performance integer division by a constant.
 - ``05_packed_load_store`` - test ``packed_[load|store]_active()`` stdlib functions performance.
 - ``06_math`` - test math functions performance.
+- ``07_loop_unroll_varying-cpu`` - test unrolling performance for varying index loop on CPU.
```

Comment:
Any specific reason to mention that it's for CPU? Benchmarks are for CPU only at the moment. If they are ported to GPU, this should work for GPU as well, right? I.e. nothing specifically bound to CPU in these test. If you agree, let's remove "CPU" from here and from test names.